packages/okta-angular/src/okta/services/okta.service.ts

```diff
@@ -183,13 +192,13 @@ export class OktaAuthService {
      * Parses the tokens from the callback URL.
      */
     async handleAuthentication(): Promise<void> {
-      const res = await this.oktaAuth.token.parseFromUrl();
+      const res = await this.token.parseFromUrl();
       const tokens = res.tokens;
       if (tokens.accessToken) {
-        this.oktaAuth.tokenManager.add('accessToken', tokens.accessToken as AccessToken);
+        this.tokenManager.add('accessToken', tokens.accessToken as AccessToken);
```

Comment:
Should we export the token keys from okta-auth-js?    

Shuo's other PR makes these keys configurable, which would break this.

Reply:
Looks like this PR is only target to auth-js@4.0 instead of v4.1.

Reply:
>         "@okta/okta-auth-js": "^4.0.0",

That includes 4.1, right?

My question about exporting the keys stands, either way.

Reply:
I could modify the semver to `~4.0.0` but there **should be no breaking changes in 4.1**, right?

Reply:
@aarongranick-okta Right.   I'm not saying you should change anything, I'm saying 4.1 should be free of breaking changes.